### PR TITLE
Create get-started-branching.md

### PR DESCRIPTION
**Pull request by danieltprice**
Get started topic for branching
https://websitemain62807-dpricebranchinggetstarted.gtsb.io/docs/get-started-with-neon/get-started-branching/

### CODE REVIEW

**Review by klink — APPROVED** [review]
Looks good! Maybe in the future we can add a video tutorial for this, too?

File: content/docs/get-started-with-neon/get-started-branching.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - **LSN**: Creates a branch with data up to the specified [Log Sequence Number (LSN)](../../reference/glossary/#lsn).
+7. Click **Create Branch** to create your branch.
+
+You are directed to the **Branches** tab where you are shown the details for your new branch.
+
+## View branches
+
+To view the branches in a Neon project:
+
+1. In the Neon Console, select a project from the project drop-down list.
```
Comment (stepashka) [inline content/docs/get-started-with-neon/get-started-branching.md:40]:
```suggestion
1. In the Neon Console, select a project from the project list.
```
we're considering getting rid of that dropdown, so it's best to refer to the list of the projects that's not going anywhere

File: content/docs/get-started-with-neon/get-started-branching.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+The **Branches** widget on the project **Dashboard** also lists the branches in a Neon project. Selecting **Manage** from the **Branches** widget directs you to the **Branches** page, where you can view and manage branches.
+
+## Connect to a branch
+
+Now that you have created a branch, how do you connect to it? You can connect to a branch using a connection string, as you would when connecting to any Neon database. The following steps describe how to connect to a branch using `psql`.
+
+_**Note:**_ You can also query a branch from the Neon SQL Editor. For instructions, see [Query with Neon's SQL Editor](../query-with-neon-sql-editor).
+
+1. On the Neon Console, select a project from the project drop-down list at the top of the console.
```
Comment (stepashka) [inline content/docs/get-started-with-neon/get-started-branching.md:52]:
```suggestion
1. On the Neon Console, select a project from the project list in the console.
```

File: content/docs/get-started-with-neon/get-started-branching.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+2. On the project **Dashboard**, under **Connection Details**, select the branch, the database, and the user you want to connect with.
+![Connection details widget](./images/connection_details.png)
+3. Copy the connection string. A connection string includes your user name, endpoint name, and database name. The endpoint name is the name of the compute instance associated with the branch.
+5. Add your password to the connection string as shown below, and connect with `psql`. You can connect using the same user and password that you use to connect to the parent branch.
```
Comment (stepashka) [inline content/docs/get-started-with-neon/get-started-branching.md:56]:
```suggestion
5. Add your password to the connection string as shown below, and connect with `psql`. You can connect using the same user and password that you use to connect to the parent branch.
```

we also provide a 'host' part of the connection string on the branch page.
let's include this flow in the description?

File: content/docs/get-started-with-neon/get-started-branching.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+## Connect to a branch
+
+Now that you have created a branch, how do you connect to it? You can connect to a branch using a connection string, as you would when connecting to any Neon database. The following steps describe how to connect to a branch using `psql`.
+
+_**Note:**_ You can also query a branch from the Neon SQL Editor. For instructions, see [Query with Neon's SQL Editor](../query-with-neon-sql-editor).
+
+1. On the Neon Console, select a project from the project drop-down list at the top of the console.
+2. On the project **Dashboard**, under **Connection Details**, select the branch, the database, and the user you want to connect with.
+![Connection details widget](./images/connection_details.png)
+3. Copy the connection string. A connection string includes your user name, endpoint name, and database name. The endpoint name is the name of the compute instance associated with the branch.
```
Comment (stepashka) [inline content/docs/get-started-with-neon/get-started-branching.md:55]:
```suggestion
3. Copy the connection string. A connection string includes your user name, endpoint address, and database name. The endpoint is the compute instance associated with the branch.
```

File: content/docs/get-started-with-neon/get-started-branching.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+If you want to connect to a branch from an application, the **Connection Details** widget on the project **Dashboard** also provides connection examples for various languages and frameworks.
+
+## Delete a branch
+
+Deleting a branch is a permanent action. Deleting a branch also deletes the branch endpoint, which is the compute instance associated with the branch.
+
+To delete a branch:
+
+1. In the Neon Console, select a project from the project drop-down list.
```
Comment (stepashka) [inline content/docs/get-started-with-neon/get-started-branching.md:70]:
```suggestion
1. In the Neon Console, select a project from the project list.
```

File: content/docs/get-started-with-neon/get-started-branching.md (hunk 3 above)
Comment (danieltprice) [reply content/docs/get-started-with-neon/get-started-branching.md:56]: Added mention of endpoint hostname.

**Review by stepashka — APPROVED** [review]

**Pull request merged**